cmd/influx_inspect/buildtsi/buildtsi.go

```diff
@@ -386,7 +390,13 @@ func (cmd *Command) collectTSMFiles(path string) ([]string, error) {
 	return paths, nil
 }
 
-func (cmd *Command) collectWALFiles(path string) ([]string, error) {
+func collectWALFiles(path string) ([]string, error) {
+	if path == "" {
+		return []string{}, os.ErrNotExist
```

Comment:
It's rarely correct to return an allocated slice with an error. This should just be `return nil, os.ErrNotExist`. Same below.